test/v1/summary.test.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) 2002-2019 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import neo4j from '../../src/v1';
+import sharedNeo4j from '../internal/shared-neo4j';
+
+describe('result summary', () => {
+
+  let driver, session;
+
+  beforeEach(done => {
+    driver = neo4j.driver('bolt://localhost', sharedNeo4j.authToken);
+    session = driver.session();
+
+    session.run("MATCH (n) DETACH DELETE n").then(done);
+  });
+
+  afterEach(() => {
+    driver.close();
+  });
+
+  it('should get result summary', done => {
+    // When & Then
+    session.run("CREATE (p:Person { Name: 'Test'})").then(result => {
+      var summary = result.summary;
```

Comment:
Imho it is better to always use `let` or `const` instead of `var` because they have a narrower scope.